goinverse-oss/app
# Implement loading and welcome screens

## Description
- Shows welcome screen on first launch
- Allows playback of audio samples

Dev note: if you want to see this again after the first launch, open Redux Dev Tools and dispatch this action:
```json
{
  "type": "theliturgists/app/welcome/SET_SHOW_WELCOME",
  "payload": true
}
```

TODO:
- [x] Get actual meditation sample audio (currently just the podcast sample mp3)
- [x] Get and insert testimonials

## Motivation and Context
Closes #58.

## How Has This Been Tested?
- Launch app, play audio, step through slides, proceed to main app.
- Reload app; on second launch, welcome does not appear.
